mono/metadata/reflection.c

```diff
@@ -486,6 +486,17 @@ mono_type_get_object_checked (MonoDomain *domain, MonoType *type, MonoError *err
 	return res;
 }
 
+MonoReflectionTypeHandle
+mono_type_get_object_handle (MonoDomain *domain, MonoType *type, MonoError *error)
+{
+	/* NOTE: We happen to know that mono_type_get_object_checked returns
```

Comment:
If you reference this comment from `mono_type_get_object_checked`, it could help avoid future breakage here.